smartcontractkit/external-adapters-js
# Update Synthetix Debt Pool EA to pull contract addresses using AddressResolver

## Closes33284

## Description

Update Synthetix Debt Pool EA to pull contract addresses using AddressResolver

## Changes

- Remove `synthetixjs/contracts-interface` depedency
- Read contract addresses from `AddressResolver`



## Steps to Test

1. Run EA and ensure successful responses
2. Manually verify result by looking at contract values
3. Write integration tests

## Quality Assurance
